cypress/integration/report_page_test.js

```diff
@@ -0,0 +1,105 @@
+        cy.get(".toc").within(() => {
+            cy.contains("Data").click();
+            cy.contains("Exons").click();
+        })
+
+        cy.get('.report-item-title').contains("Exons")
+        .parent().parent(".report-item")
+        .as('exonsTable');
+        cy.wait(100); 
+        cy.isInViewport("@exonsTable");
+    });
+
+    it("can collapse and expand a section on report page", function(){
+        cy.get('.report-table-heading').contains("Data")
+        .parent(".report-table")
+        .as('dataSection');
+
+        cy.get("@dataSection").within(() => {
+            cy.get(".report-item-title").should("exist");
+            cy.get(".icon-chevron-up").click();
+            cy.get(".report-item").should("not.exist");
+        })
+    });
+
+    it("can perform a region search with its sequence", function(){
+        cy.get(".report-table").should("have.id","summary").within(() => {
+            cy.get('.report-table-link > span').click();
+        })
+        cy.url().should("include","/regions");
+    });
+
+    it("can view the sequence of a gene", function(){
+        cy.get(".report-table").should("have.id","summary").within(() => {
+            cy.get('.fasta-value > .dropdown').click();
+        })
+        cy.isInViewport(".fasta-sequence");
+    });
+
+    it("can download the sequence of a gene in FASTA format", function(){
+        cy.get(".report-table").should("have.id","summary").within(() => {
+            cy.get('.fasta-value > .fasta-download').click();
+        })
+        cy.readFile('cypress/downloads/PF14_0187.fasta').should('contain', 'PF14_0187');
+    });
+
+    it("can expand a section to view the data displayer", function(){
+        cy.get('.report-table-heading').contains("Data")
+        .parent(".report-table").find(".report-item").filter(':contains("Exons")')
+        .as('exonsSection');
+
+        cy.get("@exonsSection").within(() => {
+            cy.get('.im-table').should("exist");
+            cy.get('.report-item-toggle').click();
+            cy.get('.im-table').should("not.exist");
+        })
+    });
+
+    
+    it("can copy the permanent URL of the report page of the object", function(){
+        cy.get(".visible-md-block").within(() => {
+            cy.contains("Copy permanent URL").click();
+            cy.get(".permanent-url-container").should("be.visible").within(() => {
+                cy.get("input.form-control").should("be.visible").invoke('val').should("include","http://localhost:9999/biotestmine/gene:PF14_0187");
+                cy.get("button").contains("Copy").should("be.visible").click({multiple:true});
+            });
+        })
+        // cy.wait(500);
+        // cy.task("getClipboard").should("contain","/biotestmine/gene:PF14_0187");
+    });
+
+    it("can check if the gene is present in any list", function(){
+        cy.get(".col-lg-12").filter(':contains("Lists")').should("be.visible").as('listsSection');
```

Comment:
Can you change `.col-lg-12` to `.sidebar-entry` if it works? The col classes are for layout and may change.